### PR TITLE
SAAS-548 update profile tests

### PR DESCRIPTION
**Pull request by yurkovychv**
[SAAS-561 PR](https://github.com/percona-platform/saas-ui/pull/96) is included here.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -73,6 +73,7 @@
     "@testing-library/cypress": "^7.0.4",
     "@types/faker": "^5.1.2",
     "@types/react-redux": "^7.1.9",
+    "axios": "^0.21.1",
```
Comment (nicolalamacchia) [inline package.json:76]: @yurkovychv why do we need `axios`? Isn't it possible to use `fetch` as we already do?

Comment (yurkovychv) [reply package.json:76]: @nicolalamacchia I think I don't get your point. There is no `fetch` in dependencies. Could you point me to where `fetch` is being used?

**Review by puneet0191 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (nicolalamacchia) [reply package.json:76]: `fetch` is native, no need for a library :)

Comment (yurkovychv) [reply package.json:76]: does not work with node application) that's why I asked )

Comment (nicolalamacchia) [reply package.json:76]: oh man, you're right, sorry for the confusion 🤦 I'm too used to front end code 😅

**Review by nicolalamacchia — APPROVED** [review]

File: cypress/plugins/oktaRequest.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+const axios = require('axios');
+
+export const oktaRequest = async ({baseUrl, urlSuffix, method = 'get', token, data = {}}) => {
+  let response;
+
+  try {
```
Comment (atymchuk) [inline cypress/plugins/oktaRequest.js:6]: If we leverage a promise syntax here, i.e. `.then()` and `.catch`, it could look much cleaner.

**Review by atymchuk — APPROVED** [review]

File: cypress/plugins/oktaRequest.js (hunk 2 above)
Comment (yurkovychv) [reply cypress/plugins/oktaRequest.js:6]: thanks, will do next time

**Pull request merged**